content/en-us/_index.md

```diff
@@ -4,14 +4,14 @@ title: "Home"
 {{< atlas-logo >}} 
 # Welcome
 
-Welcome to the IoT Atlas, where successful journeys using the internet of things rely on maps.
+Welcome to the IoT Atlas, where successful journeys using the Internet of Things (IoT) rely on maps.
 
-The most successful Internet of Things (IoT) projects have clearly defined business outcomes. The vast majority of IoT projects traverse different phases, starting from prototype, through pilot and limited production toward production, to achieve the identified business outcome at scale. The IoT Atlas supports your project by explaining the why, what, and who of commonly used, modern IoT designs.
+The most successful IoT projects have clearly defined business outcomes. The vast majority of IoT projects traverse different phases, starting from prototype, through pilot and limited production toward production, to achieve the identified business outcome at scale. The IoT Atlas supports your project by explaining the why, what, and who of commonly used, modern IoT designs.
```

Comment:
After a quick chat among the moderators, we agree that changing IoT after the initial mention is more clear.